### PR TITLE
Build home screen user interface

### PR DESCRIPTION
**Pull request by dakinnir**

### CODE REVIEW

File: SwChat/Extensions.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
+//
+//  Extensions.swift
+//  SwChat
+//
+//  Created by Daniel Akinniranye on 2/21/22.
+//
+
+import Foundation
+import SwiftUI
+
```
Comment (ekareem) [inline SwChat/Extensions.swift:10]: are all extensions going to go to this file? or are we going to separate extension files for each class we extend

File: SwChat/Resources/Assets.xcassets/app-logo.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (ekareem) [inline SwChat/Resources/Assets.xcassets/app-logo.imageset/Contents.json:1]: The size of that logo should be close to the image size as much as possible. currently it's too big Imo too much negative space

Comment (dakinnir) [reply SwChat/Resources/Assets.xcassets/app-logo.imageset/Contents.json:1]: I will fix that as soon as possible

**Review by ekareem — COMMENTED** [review]
LGTM

**Pull request merged**